features/titlebar-area.yml

```
@@ -0,0 +1,9 @@
+name: Titlebar area environment variables
+description: The `titlebar-area-*` CSS environment variables represent the header area of a Progressive Web App that is not hidden by the system's buttons. For example, ` height:\ env(titlebar-area-height)` sets the element's height to the height of the space reserved for the titlebar.
```

I think we need to mention the phrase "Window Controls Overlay" somewhere in there, for searchability. This feature is only useful if the PWA is a desktop one, only if it is actually installed, only if the manifest defines the `window-controls-overlay` display-mode, and only if the user actually opted-in to that display mode after having installed the app.
Only then do the `titlebare-area-*` variables have non-0 values.

We clearly can't say all of this in there, how about this:

```suggestion
description: The `titlebar-area-*` CSS environment variables represent the area that the titlebar of a PWA that uses the `window-controls-overlay` display-mode normally occupies. For example, `height:\ env(titlebar-area-height)` sets the element's height to the height of the space normally reserved for the titlebar.
```

That said, how about this different direction:

- We keep `env()` and `safe-area-inset` as one feature.
- We create a second feature called `window-controls-overlay`, which includes the new `titlebar-area-*` variables + the other WCO-related bits (Navigator.windowControlsOverlay, WindowControlsOverlay, WindowControlsOverlayGeometryChangeEvent, ...)

---

The separate WCO feature is in #1762